scripts/tools/benchmark_feature_extractors.py

```
@@ -0,0 +1,413 @@
+    # Save results to a JSON file
+    uv run python scripts/tools/benchmark_feature_extractors.py --out output/bench_fe.json
+
+    # CPU-only (useful for CI where no GPU is available)
+    uv run python scripts/tools/benchmark_feature_extractors.py --device cpu
+
+Results are printed in a Markdown table and optionally written as JSON so
+they can be included in context notes or CI artefacts.
+
+Observation space used (matches the default robot environment)
+--------------------------------------------------------------
+- ``rays``:        shape (1, 272) — one timestep × 272 LiDAR rays
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)